src/nyc_trees/apps/home/routes.py

```diff
@@ -19,3 +21,7 @@
 
 training_list_page = route(GET=do(render_template('home/training.html'),
                                   v.training_list_page))
+
+groups_to_follow = route(GET=do(render_template('home/groups_to_follow.html'),
+                                update({'chunk_size': 2}),
```

Comment:
`chunk_size` is an optional param, and the vanilla `group_list` view (from a different app) does not provide it. In this case we want to reuse everything about that view by importing it but stick something else in the context, which I think is done most easily in the more powerful route layer. Does that make sense?